doc/usage.md

```diff
@@ -158,6 +157,20 @@ made to the copy-on-write image, and not to the backing file. Only the
 modified data will use actual disk space.
 A copy-on-write image can be created using `cp -c` or [clonefile(2)](http://www.manpagez.com/man/2/clonefile/).
 
+#### Cloud-init
+
+The `--device virtio-blk` option could also be used to supply an initial configuration to cloud-init through an disk image.
+
+The ISO image file must be labelled CIDATA and it must contain the user-data and meta-data files. 
+It is also possible to add further configurations by using the network-config and vendor-data files.
+See https://cloudinit.readthedocs.io/en/latest/reference/datasources/nocloud.html#runtime-configurations for more details.
+
+To create the ISO image you could use the following command within a folder containing the user-data and meta-data files
+```
+mkisofs -output seed.img -volid cidata -rock {user-data,meta-data}
```

Comment:
`cidata` is apparently case insensitive since you used the upper case version before and now it is lower case?

Reply:
yes the name can be cidata or CIDATA. I thought I had written it, updating. Reference -> https://cloudinit.readthedocs.io/en/20.4.1/topics/datasources/nocloud.html